tests/v2/validation/workloads/standalone_workload_test.go

```diff
@@ -0,0 +1,140 @@
+//go:build (validation || infra.any || cluster.any || sanity) && !stress && !extended
+
+package workloads
+
+import (
+	"testing"
+
+	"github.com/rancher/shepherd/clients/rancher"
+	management "github.com/rancher/shepherd/clients/rancher/generated/management/v3"
+	"github.com/rancher/shepherd/extensions/clusters"
+	"github.com/rancher/shepherd/pkg/session"
+	log "github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type WorkloadTestSuite struct {
+	suite.Suite
+	client  *rancher.Client
+	session *session.Session
+	cluster *management.Cluster
+}
+
+func (w *WorkloadTestSuite) TearDownSuite() {
+	w.session.Cleanup()
+}
+
+func (w *WorkloadTestSuite) SetupSuite() {
+	w.session = session.NewSession()
+
+	client, err := rancher.NewClient("", w.session)
+	require.NoError(w.T(), err)
+
+	w.client = client
+
+	log.Info("Getting cluster name from the config file and append cluster details in connection")
```

Comment:
Is this necessary? I'd like to remove the logging messages in general in `SetupSuite`.

Reply:
These log messages where part of the original code. My goal was not really to change the original code in this PR. I'll probably clean up the code a bit after POC talks are finished.